src/database/schema/ChatSchema.ts

```diff
@@ -313,19 +352,27 @@ class ChatSchema implements BaseDbSchema {
       user: null,
       userId,
       isMe: true,
-      isContinuous: true
+      isContinuous: true,
+      attachmentJson: JSON.stringify(attachments)
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `attachmentJson` is not typed here. It would be better to use a specific type instead of `any`.

```diff
-    attachmentJson: JSON.stringify(attachments)
+    attachmentJson: JSON.stringify(attachments as AttachmentInterface[])
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->